UnitTests/GitUITests/CommandsDialogs/FormFileHistoryControllerTests.cs

```diff
@@ -30,10 +30,9 @@ public void TryGetExactPathName_Should_return_null_on_not_existing_file(string p
         }
 
         [TestCase(@"c:\Users\Public\desktop.ini")]
-        [TestCase(@"c:\pagefile.sys")]
         [TestCase(@"c:\Windows\System32\cmd.exe")]
         [TestCase(@"c:\Users\Default\NTUSER.DAT")]
-        [TestCase(@"c:\Program Files (x86)\Microsoft.NET\Primary Interop Assemblies")]
+        [TestCase(@"c:\Program Files (x86)\microsoft.net\primary interop assemblies")]
```

Comment:
This is not a correct fix for the test.

Reply:
What is the correct fix? 
AppVeyor has lowercase path .